.github/workflows/e2e-test-beta4-dev.yml

```diff
@@ -0,0 +1,19 @@
+name: E2E Test Beta-4
+
+on:
+  schedule:
+    - cron: '* * * * *'
```

Comment:
Hmm, I don't know how GitHub will do that each minute, but we can merge and see=)